lib/httpotion.ex

```diff
@@ -248,16 +250,18 @@ defmodule HTTPotion.Base do
       def handle_response(response) do
         case response do
           { :ok, status_code, headers, body, _ } ->
+            headers1 = process_response_headers(headers)
```

Comment:
Resist the Erlang! You can rebind `headers` here rather than using versioned variable names.

Reply:
Yeah but we know what it's actually doing so rebinding feels dirty... but hey, that's why I asked if @myfreeweb has a preference, I can switch to rebinding if preferred.

Reply:
Something like `processed_headers` would be better than `headers1`. Rebinding is fine too I guess.